todocore/src/main/java/com/knighttodo/todocore/service/DayTodoService.java

```diff
@@ -90,5 +90,17 @@ public DayTodoVO updateIsReady(UUID dayId, UUID dayTodoId, boolean isReady) {
         dayTodoVO = dayTodoMapper.toTodoVO(dayTodoRepository.save(dayTodoMapper.toTodo(dayTodoVO)));
         return experienceService.calculateExperience(dayTodoVO);
     }
+
+    @Transactional
+    public DayTodoVO updateDayTodoById(UUID dayTodoId, Integer newDayTodoOrderNumber) {
+        DayTodoVO dayTodoVO = findById(dayTodoId);
+        dayTodoVO.setOrderNumber(newDayTodoOrderNumber);
+        DayTodo dayTodo = dayTodoRepository.save(dayTodoMapper.toTodo(dayTodoVO));
+        return dayTodoMapper.toTodoVO(dayTodo);
+    }
+
+    public boolean isExistDayTodo(UUID dayTodoId) {
+        return dayTodoRepository.findByIdAlive(dayTodoId).isPresent();
```

Comment:
I don't see the point in adding this additional call to DB as well

Reply:
Ok. I deleted this method.